backend/src/main/kotlin/org/loculus/backend/controller/SubmissionController.kt

```diff
@@ -108,12 +116,15 @@ open class SubmissionController(
                 innerDataUseTermsType = dataUseTermsType
             }
         }
-
+        val fileMappingParsed = fileMapping?.let {
+            objectMapper.readValue(it, object : TypeReference<SubmissionIdFilesMap>() {})
```

Comment:
We should probably check in the backend that the submitted fileCategories are allowed fileCategories - the objectMapper doesn't do this, as it only tread `fileCategory` as a `String`

Reply:
yes, we have an issue for that in #3994

Reply:
resolving this for now